lib/raidenclient/RaidenClient.ts

```diff
@@ -87,6 +104,20 @@ class RaidenClient extends BaseClient {
     }
   }
 
+  /**
+   * Subscribe to channel events for all existing channels.
+   */
+  public async init() {
+    if (!this.isDisabled()) {
+      const channels = await this.getChannels();
+      const promises: Promise<void>[] = [];
+      channels.forEach((channel) => {
+        promises.push(this.subscribeChannelEvents(channel.channel_address));
+      });
+      await Promise.all(promises);
```

Comment:
I believe this actually returns a stream that's supposed to stay open as long as the channel is open, but it might be a good idea to try to reestablish the stream if it closes.

Reply:
Actually upon rereading the documentation, I think you may be right that this just returns a chunk of data and closes the stream. I'll try to test that endpoint and rework this if that's confirmed, but it's certainly inconvenient if so because we'd have to make repeated requests when we are the maker for each pending swap to find out its resolution.